README.md

```diff
@@ -265,7 +267,15 @@ The following targets are currently supported:
 ```json
 {
   "dotnet": {
-    "namespace": "Acme.Hello"
+    /* required */
+    "namespace": "Acme.Hello",
+    "packageId": "Acme.Hello",
+
+    /* optional */
+    "title": "ACME Hello",
+    "iconUrl": "path/to/icon.svg",
```

Comment:
Add comments with defaults to all fields

Reply:
✔️